test/models/mainstream_browse_page_test.rb

```diff
@@ -0,0 +1,197 @@
+    it "returns the browse page base_path" do
+      assert_equal "/browse/benefits/child", @page.base_path
+    end
+
+    it "returns the browse page title" do
+      assert_equal "Child Benefit", @page.title
+    end
+
+    it "returns the browse page description" do
+      assert_equal "Information about eligibility, claiming and when Child Benefit stops", @page.description
+    end
+  end
+
+  [
+    "top_level_browse_pages",
+    "second_level_browse_pages",
+  ].each do |link_type|
+    describe link_type do
+      it "returns the title, base_path and description for all linked items" do
+        @api_data["links"][link_type] = [
+          {
+            "title"=>"Foo",
+            "description" => "All about foo",
+            "base_path"=>"/browse/foo",
+          },
+          {
+            "title"=>"Bar",
+            "description" => "All about bar",
+            "base_path"=>"/browse/bar",
+          },
+        ]
+
+        items = @page.public_send(link_type)
+
+        assert_equal 'Foo', items[0].title
+        assert_equal '/browse/bar', items[1].base_path
+        assert_equal 'All about foo', items[0].description
+      end
+
+      it "returns empty array with no items" do
+        assert_equal [], @page.public_send(link_type)
+      end
+
+      it "returns empty array when the links field is missing" do
+        @api_data.delete("links")
+        assert_equal [], @page.public_send(link_type)
+      end
+    end
+  end
+
+  describe "active_top_level_browse_page" do
+    it "returns the title, base_path and description for the linked item" do
+      @api_data["links"]["active_top_level_browse_page"] = [{
+        "title"=>"Foo",
+        "description" => "All about foo",
+        "base_path"=>"/browse/foo",
+      }]
+
+      assert_equal 'Foo', @page.active_top_level_browse_page.title
+      assert_equal '/browse/foo', @page.active_top_level_browse_page.base_path
+      assert_equal 'All about foo', @page.active_top_level_browse_page.description
+    end
+
+    it "returns nil with no items" do
+      assert_nil @page.active_top_level_browse_page
+    end
+
+    it "returns nil when the links field is missing" do
+      @api_data.delete("links")
+      assert_nil @page.active_top_level_browse_page
+    end
+  end
+
+  describe "related_topics" do
+    it "returns the title, base_path and description for all related topics" do
+      @api_data["links"]["related_topics"] = [
+        {
+          "title"=>"Foo",
+          "description" => "All about foo",
+          "base_path"=>"/browse/foo",
+        },
+        {
+          "title"=>"Bar",
+          "description" => "All about bar",
+          "base_path"=>"/browse/bar",
+        },
+      ]
+
+      assert_equal 'Foo', @page.related_topics[0].title
+      assert_equal '/browse/bar', @page.related_topics[1].base_path
+      assert_equal 'All about foo', @page.related_topics[0].description
+    end
+
+    #it "returns empty array with no items" do
```

Comment:
I was on the fence here.  These will replace the block below when we're no longer supporting detailed guide categories from whitehall.